docs/icp/17-box-shadow-hover/17-box-shadow-hover.md

```diff
@@ -133,7 +135,276 @@ box-shadow: 5px 5px 5px 2px #808080;
\ No newline at end of file
+When you view this HTML page in a browser, the image will have a box shadow applied to it according to the styles defined by the `.img-tiger` class. When you move your mouse over the image, the border-radius will change to 20px with a transition effect, as defined by the `.img-tiger:hover` selector.
+
+## Background image
+
+The `background-image` property is used to set one or more background images.
+
+**Syntax :**
+
+```css
+background-image: url("");
+```
+
+**Example :**
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>background-image</title>
+    <style>
+      .container {
+        border: 1px solid black;
+        width: 600px;
+        height: 400px;
+        background-image: url("https://images.unsplash.com/photo-1579546929518-9e396f3cc809?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8fA%3D%3D&w=1000&q=80");
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>This is a card</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Optio
+        voluptates cum veritatis odio voluptas nihil commodi sunt facere illum
+        quis nostrum nam quasi aut ipsam aliquam, similique accusamus nulla!
+        Molestias.
+      </p>
+    </div>
+  </body>
+</html>
+```
+
+**Output :**
+
+<img src="/icp/17/step-4.png" alt="step-4" width="600px"/>
+
+### Background-size
+
+The background-size property is used for the size of background images.
+
+### `backgound-size: auto`
+
+This property used to set images to their original size.
+
+**Example :**
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>background-image</title>
+    <style>
+      .container {
+        border: 1px solid black;
+        width: 600px;
+        height: 400px;
+        background-image: url("https://cdn-icons-png.flaticon.com/128/3595/3595455.png");
+        background-size: auto;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>This is a card</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Optio
+        voluptates cum veritatis odio voluptas nihil commodi sunt facere illum
+        quis nostrum nam quasi aut ipsam aliquam, similique accusamus nulla!
+        Molestias.
+      </p>
+    </div>
+  </body>
+</html>
+```
+
+**Output :**
+
+<img src="/icp/17/step-5.png" alt="step-4" width="600px"/>
+
+**Example explanation:**
+
+In the above example, we set background image by using `background-img` property. Also we can set `background-size` that sets the original size of the image and image is small that's why it display image repetedly.
+
+### `background-size: cover`
+
+This property is used to cover the area of the container.
```

Comment:
This value is used to scale the background image to cover the entire container, while maintaining the image's aspect ratio. The image may be cropped or stretched as needed.